lib/drivers/insulet/insuletDriver.js

```diff
@@ -461,6 +462,12 @@ module.exports = function (config) {
     eLogRecSizeErr: { value: 8, name: 'eLogRecSizeErr' }
   };
 
+  var TEMP_BASAL_TYPES = {
```

Comment:
Should we add an "enum" somewhere common to standardise `percent` and `Units/hour`?

Reply:
This one is specific to OmniPod though, as other pumps may have them in a different order.

Reply:
I meant more something like:
```
const TEMP_BASAL_TYPES = {
  OFF: 'off',
  PERCENT: 'percent',
  UNITS: 'Units/hour',
};
```
in a common place, so that in our driver code we can use, say, `TEMP_BASAL_TYPES.UNITS`, instead of `'Units/hour'`, so that we don't end up with typos. I guess that it's not that big of a win in JavaScript, because there's no type-safety and compile-time checks.